### PR TITLE
Database Sync Worker Example Branch - Start

### PR DESCRIPTION
**Pull request by improbable-davidrodriguez**
Changed the Example Project to have a dummy version of storing scores with persistence.

### CODE REVIEW

File: spatial/provided/default_launch.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
+{
+  "template": "small",
```
Comment (m-samiec) [inline spatial/provided/default_launch.json]: Can you change this for the updated template value?

File: Game/Source/GDKShooter/Private/Game/Components/DeathmatchScoreComponent.cpp [hunk 2]
```diff
@@ -25,6 +25,9 @@ void UDeathmatchScoreComponent::RecordNewPlayer(APlayerState* PlayerState)
 		NewPlayerScore.PlayerName = PlayerState->GetPlayerName();
 		NewPlayerScore.Kills = 0;
 		NewPlayerScore.Deaths = 0;
+		
+		NewPlayerScore.AllTimeKills = 0;	// TODO: Get this value from persistent storage
```
Comment (m-samiec) [inline Game/Source/GDKShooter/Private/Game/Components/DeathmatchScoreComponent.cpp]: Can we change these TODOs to TUTORIAL-TODOs?

File: spatial/provided/default_launch.json [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  "world": {
+    "dimensions": {
+      "x_meters": 2000,
+      "z_meters": 2000
+    },
+    "chunk_edge_length_meters": 50,
+    "streaming_query_interval": 4,
+    "legacy_flags": [
```
Comment (m-samiec) [inline spatial/provided/default_launch.json]: These legacy flags are outdated, could they be resynced with the defaults in this project?

**Comment by peterimprobable** [comment]
Is the database worker work being included in the release branch?
My understanding was that it was a tutorial branch.

**Pull request closed without merging**